tests/ops/api/v1/endpoints/test_privacy_notice_endpoints.py

```diff
@@ -37,6 +30,7 @@
     PrivacyNoticeCreation,
     PrivacyNoticeResponse,
 )
+from fides.api.util.consent_util import validate_notice_data_uses
```

Comment:
nit: may make sense to move `TestValidateDataUses` test class over to `test_consent_util.py` now that the underlying utility function has been moved to `consent_util.py`?

Reply:
that makes sense, thanks